templates/python/LocalPubsub/README.md

```diff
@@ -0,0 +1,13 @@
+## My Project
```

Comment:
Maybe add a description here on how to install it and run the tests or what this templated does